zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/DbRoleState.java

```diff
@@ -82,4 +96,11 @@ public long getRoleKeyByName(final String roleName) {
     final var fkRoleKey = roleByNameColumnFamily.get(this.roleName);
     return fkRoleKey != null ? fkRoleKey.inner().getValue() : -1;
   }
+
+  @Override
+  public EntityType getEntityType(final long roleKey, final long entityKey) {
+    this.roleKey.wrapLong(roleKey);
+    this.entityKey.wrapLong(entityKey);
+    return entityTypeByRoleColumnFamily.get(fkRoleKeyAndEntityKey).getEntityType();
```

Comment:
❌ This could result in an NPE. Let's return an `Optional<EntityType>` in this method